packages/ListBox/src/index.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ListBox, { propTypes, defaultProps } from "./ListBox";
+import { GetTypeOfChildren } from "./helpers/GetTypeOfChildren";
+import Divider from "./components/Divider";
+import Filter from "./components/Filter";
+import Footer from "./components/Footer";
+import Option from "./components/Option";
+import Popover from "./components/Popover";
+import RawItem from "./components/RawItem";
+import Trigger from "./components/Trigger";
+import Provider from "./store/Provider";
+
+const ListBoxWithProvider = React.forwardRef((props, ref) => {
+  const { children, ...moreProps } = props;
+
+  /*
+  Assures the structure of the children is one of the following:
+
+  <React.Fragrment>
+    <ListBox.Option />
+  </React.Fragrment>
+
+  - OR -
+    [
+      <React.Fragrment>
+        <ListBox.Option />
+      </React.Fragrment>,
+      <React.Fragrment>
+        <ListBox.Option />
+      </React.Fragrment>
+    ]
+    */
+
+  const extractedChildren = React.Children.map(children, child => {
+    return child !== null && React.Fragment === child.type ? child.props.children : child;
+  });
+
+  const { Filter, Footer, Options, Popover, Trigger } = GetTypeOfChildren(extractedChildren);
+
+  return (
+    <Provider {...moreProps} childrenOptions={Options}>
+      <ListBox {...moreProps} Filter={Filter} Footer={Footer} Popover={Popover} Trigger={Trigger} ref={ref}>
+        {Options}
+      </ListBox>
+    </Provider>
+  );
+});
+
+export default ListBoxWithProvider;
+
+ListBoxWithProvider.propTypes = propTypes;
+ListBoxWithProvider.defaultProps = defaultProps;
+ListBoxWithProvider.Divider = Divider;
+ListBoxWithProvider.Filter = Filter;
+ListBoxWithProvider.Footer = Footer;
+ListBoxWithProvider.Option = Option;
+ListBoxWithProvider.Popover = Popover;
```

Comment:
Well, the thing is not a body, is actually a Popover, and if you as a consumer learned how to use the Popover you can transfer your knowledge in using it here, the same props are forwarded to the original Popover. If I change the name we lose the ability to transfer that intuitiveness. 

Also changing the name to` ListBox.Body` will be misleading because when the Popover  `is inline` there is not a Popover rendered then there will not be a "Body".

Reply:
Hmm but for the inline case, it doesn't make sense to call it Popover either as a popover is by definition not inline?

Reply:
`when the Popover is inline there is not a` 
should read 
`when the **LISTBOX**  is inline there is not a Popover`